packages/pds/tests/seeds/basic.ts

```diff
@@ -29,7 +29,7 @@ export default async (sc: SeedClient, mq?: MessageQueue) => {
   )
   const img2 = await sc.uploadFile(
     carol,
-    'tests/image/fixtures/key-portrait-small.jpg',
+    'tests/image/fixtures/key-alt.jpg',
```

Comment:
I wanted to give carol a unique file that nobody else had uploaded. This changed a couple of test snapshots as well